public/vis_controller.ts

```diff
@@ -62,48 +62,48 @@ export function getEnhancedTableVisualizationController(
-          // - table_vis_fn => unserialize the params and prepare them for the final React/Angular bridge
-          // - visualization_renderer => creates the wrapper component for this controller and passes the params
-          //
-          // In case some prop is missing check into the top of the chain if they are available and check
-          // the list above that it is passing through
-          this.$scope.vis = this.vis;
-          this.$scope.visState = { params: visParams, title: visParams.title };
-          this.$scope.esResponse = esResponse;
+            // How things get into this $scope?
+            // To inject variables into this $scope there's the following pipeline of stuff to check:
+            // - visualize_embeddable => that's what the editor creates to wrap this Angular component
+            // - build_pipeline => it serialize all the params into an Angular template compiled on the fly
+            // - table_vis_fn => unserialize the params and prepare them for the final React/Angular bridge
+            // - visualization_renderer => creates the wrapper component for this controller and passes the params
+            //
+            // In case some prop is missing check into the top of the chain if they are available and check
+            // the list above that it is passing through
+            this.$scope.vis = this.vis;
+            this.$scope.visState = { params: visParams, title: visParams.title };
+            this.$scope.esResponse = esResponse;
 
-          this.$scope.visParams = visParams;
-          this.$scope.renderComplete = resolve;
-          this.$scope.renderFailed = reject;
-          this.$scope.resize = Date.now();
-          this.$scope.$apply();
-        };
+            this.$scope.visParams = visParams;
+            this.$scope.renderComplete = resolve;
+            this.$scope.renderFailed = reject;
+            this.$scope.resize = Date.now();
+            this.$scope.$apply();
+          };
 
-        if (!this.$scope && this.$compile) {
-          this.$scope = this.$rootScope.$new();
-          this.$scope.uiState = handlers.uiState;
-          this.$scope.filter = handlers.event;
-          updateScope();
-          this.el.find('div').append(this.$compile(enhancedTableVisTemplate)(this.$scope));
-          this.$scope.$apply();
-        } else {
-          updateScope();
-        }
-      } catch (error) {
-        reject(error);
+          if (!this.$scope && this.$compile) {
+            this.$scope = this.$rootScope.$new();
+            this.$scope.uiState = handlers.uiState;
+            this.$scope.filter = handlers.event;
+            updateScope();
+            this.el.find('div').append(this.$compile(enhancedTableVisTemplate)(this.$scope));
+            this.$scope.$apply();
+          } else {
+            updateScope();
+          }
+        } catch (error) {
+            reject(error);
```

Comment:
Can you just remove 2 spaces on line 106 please?
And then, it will be OK for me. I will merge the PR.

Reply:
I committed a new version with this change :)